docs/release_notes.rst

```diff
@@ -4,8 +4,11 @@
 Release Notes
 =============
 
-This page contains the release notes for Qiskit, starting from the point at which the legacy
-"elements" structure was completely removed.
+This page contains the release notes for Qiskit, starting from Qiskit 0.45, the first time that Qiskit and Qiskit Terra had the same versions.
+
+These release notes get converted into Markdown files via the infrastructure at https://github.com/Qiskit/documentation, which then gets deployed to https://docs.quantum.ibm.com/api/qiskit/release-notes. Changes to these release notes will update those release notes the next time the API docs are generated.
+
+To change release notes prior to Qiskit 0.45, update the Qiskit/documentation repository directly.
```

Comment:
This appears to be a developer-facing comment - perhaps a better place would be the docs contributing guide?  Alternatively, it could be a rST comment:
```rst
This page contains [...]

..
    These release notes [...]
```

I'm just thinking on the off-chance that this text starts getting used again at some point in the future as part of the docs build - we wouldn't this to accidentally start getting published.